src/lazy/text/buffer.rs

```diff
@@ -790,7 +807,7 @@ impl<'data> TextBufferView<'data> {
                 },
             ),
             map(
-                match_and_length(Self::match_struct),
+                match_and_length(Self::match_struct_1_1),
```

Comment:
🗺️ I added `match_struct_1_1`, so it got updated here, inside `match_value_1_1`.